akka-docs/src/main/paradox/typed/replicated-eventsourcing.md

```diff
@@ -54,6 +54,22 @@ To assist in implementing the event handler the Replicated Event Sourcing detect
 
 @scala[The same API as regular `EventSourcedBehavior`s]@java[A very similar API to the regular `EventSourcedBehavior`] is used to define the logic. 
 
+### Consuming events via gRPC transport
+
+Since Akka 2.8.0 a gRPC based transport is the recommended way to set up the replication of events between the replicas.
+
+The functionality is provided through the Akka Projection gRPC module, see the details about 
+how to use it up in the @extref[Akka Projection gRPC documentation](akka-projection:grpc-replicated-event-sourcing-transport.html)
+
+Complete samples of the gRPC transport set up can be found in the Akka Projection github repository:
+
+* [Replicated shopping cart service in Scala](https://github.com/akka/akka-projection/tree/main/samples/replicated/shopping-cart-service-scala)
+* [Replicated shopping cart service in Java](https://github.com/akka/akka-projection/tree/main/samples/replicated/shopping-cart-service-java)
+
+### Consuming events via direct access to replica databases
+
+It is also possible to consume events with a direct connection to the database of each replica.
```

Comment:
This is good, but we also have a few more sections below that are not relevant for gRPC transport. Add a note or something to those?

* Direct Replication of Events
* Hot Standby
* Sharded Replicated Event Sourced entities (the multi-dc part)

Reply:
Perhaps it would be best to place everything that has with "direct access to replica databases" in a new separate page, and have this page for general content.

Reply:
I agree, let's break it out to a separate page

Reply:
I think I have pulled everything direct-transport specific out now